docs/mesos/Vagrant.md

```diff
@@ -0,0 +1,117 @@
+ * [Git](git)
+
+## Getting Started
+1. First, clone this repository and change to the Mesos vagrant directory:
+
+  ```
+  git clone https://github.com/projectcalico/calico-containers.git
+  cd calico-containers/docs/mesos/vagrant-centos
+  ```
+
+2. Then launch the Vagrant demo:
+  ```
+  vagrant up
+  ```
+
+That's it! Your Mesos Cluster is ready to use!
+
+## Log in to Vagrant machines
+
+To connect to your servers:
+
+####Linux/Mac OS X
+Run:
+
+	vagrant ssh <hostname>
+
+####Windows
+Follow instructions from https://github.com/nickryand/vagrant-multi-putty.
+
+Then, run:
+
+	vagrant putty <hostname>
+
+## Next steps
+
+With your cluster deployed, you have everything in place to run the
+[Calico Mesos Stars Demo](stars-demo/README.md), an interesting network
+policy visualizer demo that shows how Calico can secure your cluster.
+(Note that this demo is currently worked specifically for use with the
+Docker Containerizer.)
+
+Alternatively, you can follow one of our usage guides to learn how to
+launch Calico-networked tasks with either the [Docker Containerizer]
+(./UsageGuideDockerContainerizer.md) or the [Unified Containerizer]
+(UsageGuideUnifiedContainerizer.md).
+
+## Virtual Machines Info
+
+The installed virtual machines will be running with the following config:
+
+```
+.-----------------------------------------------------------------------------------.
+| Machine Type | OS     | Hostname        | IP Address     | Services               |
+|--------------|--------|-----------------|----------------|------------------------|
+| Master       | Centos | calico-mesos-01 | 172.24.197.101 | mesos-master           |
+|              |        |                 |                | etcd                   |
+|              |        |                 |                | docker                 |
+|              |        |                 |                | zookeeper              |
+|              |        |                 |                | marathon               |
+|              |        |                 |                | marathon load-balancer |
+|              |        |                 |                | calico-node            |
+|              |        |                 |                | calico-libnetwork      |
+|--------------|--------|-----------------|----------------|------------------------|
+| Agents       | Centos | calico-mesos-02 | 172.24.197.102 | mesos-agent            |
+|              |        | calico-mesos-03 | 172.24.197.103 | docker                 |
+|              |        |                 |                | calico-node            |
+|              |        |                 |                | calico-libnetwork      |
+'-----------------------------------------------------------------------------------'
+```
+
+Note that Calico is installed on the Agents so that tasks are automatically
+networked by Calico.  Calico is also installed on the Master to provide an IP
```

Comment:
Calico is also run on the Master to allow Marathon-lb route external requests to tasks with Calico IPs